src/styles/blocks/price.scss

```diff
@@ -0,0 +1,24 @@
+.price {
+  display: flex;
+}
+
+.text1 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 24px;
+  margin-left: 17px;
+  color: #616070;
```

Comment:
The font properties 'font-family', 'font-weight', 'font-style', 'font-size', 'line-height', and 'color' are repeated in both '.text1' and '.text2'. Consider creating a common class for shared styles to follow the DRY (Don't Repeat Yourself) principle.